packages/frontend/src/features/scheduler/components/SchedulerFilters.tsx

```diff
@@ -79,6 +79,7 @@ const FilterItem: FC<SchedulerFilterItemProps> = ({
                             tileTargets: undefined,
                         });
                     }}
+                    withinPortal
```

Comment:
We need the same fix for `<FilterInputComponent`

<img width="686" alt="Screenshot 2023-11-20 at 15 44 14" src="https://github.com/lightdash/lightdash/assets/9117144/00aedcfb-10b1-4066-a0a3-064aee4deae7">

Reply:
exactly! thanks 😅

Reply:
done :https://github.com/lightdash/lightdash/pull/7985/commits/e6314449726d23b5e5775c6679d88fb1c5bc1a3d